### ISSUE TITLE
Selected row style not set

### ISSUE THREAD

**Issue by maxehrhardt** [KevinVandy/mantine-react-table#386]
### mantine-react-table version

V2

### react & react-dom versions

18.3.0

### Describe the bug and the steps to reproduce it

MantineTableBodyRow styling of selected row is not set as expected.
In V1 the styling of the selected row was set automatically:
https://www.mantine-react-table.com/docs/guides/row-selection

Which is not working in V2 anymore:
https://v2.mantine-react-table.com/docs/guides/row-selection

### Minimal, Reproducible Example - (Optional, but Recommended)

V1 example:
https://www.mantine-react-table.com/docs/guides/row-selection

V2 example:
https://v2.mantine-react-table.com/docs/guides/row-selection

### Do you intend to try to help solve this bug with your own PR?

None

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by alessandrojcm** [comment]
Hi, please confirm this still happens with the latest beta

**Comment by desytech** [comment]
Hi, can confirm `--mrt-selected-row-background-color` does not work with the lastest beta.

**Actual**
- `var(--mantine-color-body)` is used for `background-color`

**Expected** e.g.
- `var(--mrt-selected-row-background-color, --mantine-color-body)`

Workaround for 2.0.0-beta.7

1. Use CSS Modules
2. Use Row Props

```js
                mantineTableBodyRowProps={({ row }) => ({
                    onClick: row.getToggleSelectedHandler(),
                    style: {
                        cursor: "pointer",
                        backgroundColor: row.getIsSelected() ? "var(--mantine-color-gray-2)" : "inherit"
                    }
                })}
```

**Comment by alessandrojcm** [comment]
Fixed in #437
